schema/manifest_backwards_compatibility_test.go

```diff
@@ -136,14 +136,14 @@ func TestBackwardsCompatibilityManifest(t *testing.T) {
 		//
 		// curl -L -H "Authorization: Bearer ..." -H \
 		// "Accept: application/vnd.docker.distribution.manifest.v2+json" \
-		// https://registry-1.docker.io/v2/library/docker/manifests/sha256:888206c77cd2811ec47e752ba291e5b7734e3ef137dfd222daadaca39a9f17bc
+		// https://registry-1.docker.io/v2/library/docker/manifests/sha256:d901d5244c94db8151415bf3a1d3a0b10e7468e4997786b01c287cbcbaeca446
```

Comment:
Have you tested this?  I expect the `curl` call will not succeed.  I think the right solution to this problem it to make these child-media-type checks optional, with something like a `strict` option.  More background in opencontainers/image-tools#66, #403, and #452.